imod/msw/copy_files.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from shutil import copy2
+from typing import cast
+
+import numpy as np
+import xarray as xr
+
+from imod.logging import logger
+from imod.logging.loglevel import LogLevel
+from imod.msw.pkgbase import MetaSwapPackage
+from imod.typing import Imod5DataDict
+
+_LOG_MESSAGE_TEMPLATE = """\
+Will not copy files {filtered}, these will be generated by iMOD Python
+instead."""
+
+
+class CopyFiles(MetaSwapPackage):
```

Comment:
Why does this class inherit from the MetaSwapPackage?
It seems like a really general class. I don't know how it is being used but is it possible to use composition over inheritance? 

If it is possible to have this class without MetaSwapPackage as its parent class then you also don't need xarray as the data container and you have a member variable that is a list of paths

Reply:
I thought about that, my main reason why I decided to inherit MetaSwapPackage is that it makes the package object function consistently to other packages. For instance, we have plans to create a ``MetaSwapModel.dump`` method, which would dump all packages to file. To have a functional MetaSWAP model, we also need to store the paths to files that need to be copied somewhere. So by wrenching this list of strings/paths in a xarray Dataset, I'm quite sure no special-cased logic is required for this specific filecopy package and keeps things stored consistently in NetCDFs, instead of some custom textfile.